frontend/src/Helpers/Props/align.ts

```diff
@@ -3,3 +3,5 @@ export const CENTER = 'center';
 export const RIGHT = 'right';
 
 export const all = [LEFT, CENTER, RIGHT];
+
+export type Align = ('left' | 'center' | 'right') & (typeof all)[number];
```

Comment:
Is the type redundancy intentional ?

These would be the same type

```suggestion
export type Align = 'left' | 'center' | 'right;
```

```suggestion
export type Align = (typeof all)[number];
```

Reply:
Same question for Kinds, Sizes, directions... etc

Reply:
Yes, the indexed strings were causing issues in some places so I extended the type, but forgot how TS resolves the types/strings in that scenario. I'll clean them up